portfolio/src/main/webapp/index.html

```diff
@@ -18,11 +18,6 @@
 
       <body>
 
-        <button onclick="getLoginStatus()">Click to update login status</button>
```

Comment:
This button will be soon replaced by another indicator of if the user is logged in or not, so I removed it